source/robot_model/test/tests/test_kinematics.cpp

```diff
@@ -148,3 +149,39 @@ TEST_F(RobotModelKinematicsTest, TestInverseKinematics) {
     EXPECT_TRUE(joint_twist.get_velocities().isApprox(test_configs[config].get_velocities()));
   }
 }
+
+TEST_F(RobotModelKinematicsTest, TestInverseGeometry) {
+  std::vector<state_representation::JointState> test_configs;
+  state_representation::JointState config1("robot", 7);
+  state_representation::JointState config2("robot", 7);
+  state_representation::JointState config3("robot", 7);
+  // Random test configurations
+  config1.set_positions({-0.059943, 1.667088, 1.439900, -1.367141, -1.164922, 0.948034, 2.239983});
+  config2.set_positions({2.648782, -0.553976, 0.801067, -2.042097, -1.642935, 2.946476, 1.292717});
+  config3.set_positions({-0.329909, -0.235174, -1.881858, -2.491807, 0.674615, 0.996670, 0.345810});
+  test_configs.push_back(config1);
+  test_configs.push_back(config2);
+  test_configs.push_back(config3);
+
+  for (std::size_t config = 0; config < test_configs.size(); ++config) {
+    state_representation::CartesianPose reference = franka->forward_geometry(test_configs[config],"panda_link8");
+    state_representation::JointPositions q = franka->inverse_geometry(reference, "panda_link8");
+    state_representation::CartesianPose X = franka->forward_geometry(q,"panda_link8");
```

Comment:
Spaces after commas between function arguments

```suggestion
    state_representation::CartesianPose reference = franka->forward_geometry(test_configs[config], "panda_link8");
    state_representation::JointPositions q = franka->inverse_geometry(reference, "panda_link8");
    state_representation::CartesianPose X = franka->forward_geometry(q, "panda_link8");
```